lib/kadai-core/src/main/java/io/kadai/spi/routing/api/TaskRoutingProvider.java

```diff
@@ -29,6 +29,10 @@
  */
 public interface TaskRoutingProvider {
 
+  static RoutingTarget newEmptyRoutingTarget() {
```

Comment:
Do we need this? Why not using "new RoutingTarget()"?

Reply:
We need this for users, so that they can create a RoutingTarget without using our RoutingTarget class. However, I would be open to let our users also use new RoutingTarget() and delete this method. Let's discuss this!